Documentation/Tutorials/ExtendNews/FormDataProvider/Index.rst

```diff
@@ -0,0 +1,76 @@
+.. _formDataProvider:
+
+================
+FormDataProvider
+================
+
+If `displayCond` and `eval` in TCA are no longer sufficient to meet certain requirements that's where
```

Comment:
formdataprovider is far more than that. I suggest something like "dynamic changes to the tca" or so